doc/Requirements Specification/Parts/Models.lyx

```diff
@@ -179,7 +179,7 @@ status open
 
 
 \backslash
-glslabel{software architecture}{architecture}
+gls{software architecture}
```

Comment:
I’d prefer `\glslink{software architecture}{architectural}` here to not say “software” two times.